styles/globals.css

```diff
@@ -7,7 +7,6 @@ body {
   background-color: black;
 
   align-items: center;
-  text-align: center;
```

Comment:
I saw that when text-align: center was removed from globals.css, we need to change in some .css files for individual text on different pages.

For example "kommande premiär" on startpage
![image](https://user-images.githubusercontent.com/89969770/167486621-f98ae07d-5e6a-4881-95b2-e1bc0f38e80b.png)

Then there is the headline on movie, ticket and contact pages (some has no styling yet, so we can fix it later).
![image](https://user-images.githubusercontent.com/89969770/167486802-7f5db0f7-f4d2-4a21-bbd9-6597fcf9156a.png)

I don´t know if we should add text-align on the diferent pages now with this ticket or if we should fix it with some ticket for styling contactpage for example, or if we should add a new ticket for this fix?

Reply:
My suggestion is that we resolve it by adding text-align in the individual css modules.